src/main/java/net/snowflake/ingest/streaming/internal/RegisterService.java

```diff
@@ -198,8 +196,13 @@ List<FlushService.BlobData<T>> registerBlobs(
             Timer.Context registerContext =
                 Utils.createTimerContext(this.owningClient.registerLatency);
 
+            for (BlobMetadata blobMetadata : blobs) {
+              blobMetadata.getBlobLatencies().setRegisterStartTimestamp(System.currentTimeMillis());
+              blobMetadata.getBlobLatencies().setFlushStartTimestamp(System.currentTimeMillis());
```

Comment:
Nit, let's set `now` as a variable and pass that down:
```
long timeStamp = System.currentTimeMillis();
blobMetadata.getBlobLatencies().setRegisterStartTimestamp(System.currentTimeMillis());
blobMetadata.getBlobLatencies().setFlushStartTimestamp(System.currentTimeMillis());
```

Reply:
pulled out currentTimeMillis to a different variable